src/devTools.js

```diff
@@ -272,20 +327,25 @@ class DevToolsEnhancer {
     const maxAge = options.maxAge || 30;
     return next => {
       return (reducer, initialState) => {
-        this.store = configureStore(
-          next, this.monitorReducer, {
-            maxAge,
-            trace: options.trace,
-            traceLimit: options.traceLimit,
-            shouldCatchErrors: !!this.sendOnError,
-            shouldHotReload: options.shouldHotReload,
-            shouldRecordChanges: options.shouldRecordChanges,
-            shouldStartLocked: options.shouldStartLocked,
-            pauseActionType: options.pauseActionType || '@@PAUSED'
-          }
-        )(reducer, initialState);
-
-        if (realtime) this.start();
+        this.store = configureStore(next, this.monitorReducer, {
+          maxAge,
+          trace: options.trace,
+          traceLimit: options.traceLimit,
+          shouldCatchErrors: !!this.sendOnError,
+          shouldHotReload: options.shouldHotReload,
+          shouldRecordChanges: options.shouldRecordChanges,
+          shouldStartLocked: options.shouldStartLocked,
+          pauseActionType: options.pauseActionType || '@@PAUSED',
+        })(reducer, initialState);
```

Comment:
this is just styling

Reply:
Please undo all styling changes to make the review easier

Reply:
fixed